### ISSUE TITLE
Extreme mimic spawn

### ISSUE THREAD

**Issue by notsuntm** [imDaniX/Dangerous-Caves-2#45]
Extreme Mimic spawn in caves. Every cave near a players home get absolutely filled with mimics after few hours of gameplay. Also the command "/dangerouscaves kill mimic" doesn't seem to work "There's no mob's type called mimic!" and no error on console.

**Screenshot:**
Mimic spawns:
https://ibb.co/rxJPb5d [.]
Command not working:
https://ibb.co/xMcT20w [.]

**Config:**
https://pastebin.com/MfDRXpbt [.]

**Additional information**
- Server version: 1.16.5
- Plugin version: Dangerous-Caves-2.2.14-SNAPSHOT
- Server core name: Tuinty

**Comment by imDaniX** [comment]
Sorry, as for now Mimic mob is still rather experimental. I'm thinking of ways dealing with it, but as for now it's disabled by default because of this exact reason.
I'll eventually fix the issue, but for now, consider disabling this mob by setting its priority to 0.

**Comment by sunalsun** [comment]
Looking forward to that update. How about setting the priority to 1? Would that minimize the spawn or would that be same?
Also how will I de-spawn already spawned mimics since the command is not working?

**Comment by imDaniX** [comment]
The problem is that mimics are not considered as a mob after transformation. The only solution I can think of right now - just use WorldEdit.
I can add a command that will scan loaded chunks and delete Mimics, but it'll take sometime because I'm not able to do so right now.
Also, as I said before, it's not recommended to turn mimics on right now. If you really want this mob on your server - go ahead tho.

**Comment by sunalsun** [comment]
Yes, I already disabled it. 
And I'll try using world edit. But the command would be handy in the future, yes.

**Comment by imDaniX** [comment]
Temporarily added a new command `/dcaves mimeremove [radius] [me|all]`. `/dcaves mimeremove 3 all` will remove all mimic chests in 3 chunk radius around all the players. Just `/dcaves mimeremove` removes mimics in 1 chunk radius around executor(around everyone if used from console)
I didn't test it - sorry, can't do that rn. At least it will not delete player's chests in any case.

**Comment by CloudeLecaw** [comment]
is it tab complpeted? :)

**Comment by imDaniX** [comment]
@CloudeLecaw nah. I could do that if you really want to, but I hope that this command will eventually be deleted...

**Comment by vdl19** [comment]
Hello, I put the last version, the command works but it doesn't seem to remove anything even if by doing the command I have according to the chosen radius of deletion infos like "Removing 2560 mimic chests" I don't see anything disappearing around me! it's the horror and I can't remake a world from 0 nor recover an old version there was too much thing built on it since the times when I had activated the minics!
What do I have to do?

![image](https://user-images.githubusercontent.com/80985545/127002201-ee5a9b99-97b8-4ae7-b50d-117453945656.png)

**Comment by vdl19** [comment]
Hello, would it be possible to have a permission to authorize my admin moderator and guide to use the command to remove the mimic, all alone I will never get out of it:
/dcaves mimeremove 0 all

**Comment by CloudeLecaw** [comment]
Currently you just have
`dangerous.caves.command` what allows you to use the whole commands of this plugin.

**Comment by vdl19** [comment]
Thank you, I will put it in place for all the Staff to try to reduce the mimic as much as possible

**Comment by vdl19** [comment]
It didn't work, the other staff accounts with the "dangerous.caves.command" permission can't run the command in OP it's ok without OP there is this line that is written once the command is run "/dcaves mimeremove 0 all" 

![image](https://user-images.githubusercontent.com/80985545/129783747-6a055748-7834-4da0-a3ca-23dd65695f19.png)

**Comment by vdl19** [comment]
Hello, thanks to this update my admin and others can use the command "/dcaves mimeremove 0 all" with the permission "dangerous.caves.command"

**Comment by CloudeLecaw** [comment]
Still would be nice to have the mimiremove in tabcomplete :)
